### PR TITLE
Adds Reloading

### PR DESCRIPTION
**Pull request by JohnOxford**
Adds 9mm 10mm 357. 44. 45. 5.56 7.62 bullets and dies
Adds a reloading press

something something gunpowder something

Since i have no fucking clue how to distinguish between fired and unfired rounds on a list we'll just make new casings require metal to build

its really fucking late and ive been working on this for a while will review in morning

### CODE REVIEW

**Comment by companoo** [comment]
travis is a massive shitlord

File: code/modules/crafting/reloading.dm [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+	icon = 'icons/obj/reloading.dmi'
+
+/obj/item/reloading/die/ninemilimeter
+	name = "9mm reloading die"
+	desc = "A sandstone reloading die with 5 slots each carved into the shape of a 9mm bullet. You vaguely can make out the letters O and X on the bottom of the brick."
+	icon_state = 'diesmall'
+
+/obj/item/reloading/die/tenmilimeter
+	name = "10mm reloading die"
+	desc = "A sandstone reloading die with 5 slots each carved into the shape of a 9mm bullet. You vaguely can make out the letters O and X on the bottom of the brick."
```
Comment (extramrdo) [inline code/modules/crafting/reloading.dm]: 10mm in the shape of a 9mm bullet, nice. Wow, the 9mm bullet is a very versatile shape!

Comment (JohnOxford) [reply code/modules/crafting/reloading.dm]: i can't denote a difference of .1 mm in a 32x32 sprite

Comment (extramrdo) [reply code/modules/crafting/reloading.dm]: desc = "... carved shape of 9mm bullet" for all of the dies

**Comment by JohnOxford** [comment]
need some sincere help on why, even with the correct path and states, that apparently it cant find reloading.dmi

**Comment by improvedname** [comment]
image

**Comment by JohnOxford** [comment]
i noticed, i dont know how to fix that. once again, the paths, icon states, icon file, icon location are all correct and the dmi is not read only and the 's around it are 's and not "s

**Comment by improvedname** [comment]
You need to use " for states not '

**Comment by UndeadNappist** [comment]
Have you talked to Oculus about how he's doing ammo crafting yet and all that?

**Comment by UndeadNappist** [comment]
What was the deleted comment and why?

**Comment by JohnOxford** [comment]
i dont even remember what it was, if it was anything but i have no idea why they would have deleted it

i haven't spoken to oculus about his PR in particular but i'm going under the context that if and when its completed we can just commit all of this out, im not super heavily invested in it

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 2]
```diff
@@ -629,6 +629,8 @@
 	var/loot2 = list(
 				/obj/item/gun/ballistic/revolver/caravan_shotgun,
 				/obj/item/storage/box/rubbershot/beanbag,
+				/obj/item/reloading/reloadingpress,
+				/obj/item/reagent_containers/glass/bottle/gunpowder,
 				""
```
Comment (UndeadNappist) [inline code/game/objects/effects/spawners/f13lootdrop.dm]: why is there a double quote here?

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 3]
```diff
@@ -640,6 +642,8 @@
 
 	var/loot4 = list(
 				/obj/item/gun/ballistic/revolver/zipgun,
+				/obj/item/reloading/reloadingpress,
+				/obj/item/reagent_containers/glass/bottle/gunpowder,
 				"",
 				""
```
Comment (UndeadNappist) [inline code/game/objects/effects/spawners/f13lootdrop.dm]: see above

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 4]
```diff
@@ -702,6 +711,7 @@
 	var/loot2 = list(
 				/obj/item/gun/ballistic/shotgun,
 				/obj/item/storage/box/lethalshot,
+				/obj/item/reloading/reloadingpress,
 				""
```
Comment (UndeadNappist) [inline code/game/objects/effects/spawners/f13lootdrop.dm]: what da zog

**Pull request closed without merging**